### DISCUSSION TITLE
Change listening port?

### DISCUSSION THREAD

**Discussion by MTGMAD** [oobabooga/text-generation-webui#998] [General]
Can anyone point my in the right direction to change the port it listens on?  It is using the exact same as AUTOMATIC1111 makes it a little difficult to run both.  Thank you

**Comment by MTGMAD** [comment]
Actually never mind, I must be tired I didn't notice they are one off

**Comment by MaiJames** [comment]
Actually you were right. Both use the 7860 by default, but if you try to run both at the same time, the second one you run changes to 7861

**Comment by bmoconno** [comment]
There is also a `--listen-port` argument that you can pass into `server.py`.

**Reply by qbxdp** [reply]
Sorry for disturbance but where does this file located?

**Reply by hayashibob** [reply]
To define persistent command-line flags like --listen or --api, edit the CMD_FLAGS.txt file with a text editor and add them there. 
Flags can also be provided directly to the start scripts, for instance, ./start-linux.sh --listen --listen-port 7861.

**Comment by wos83** [comment]
is true, thx.
Nice http://localhost:7861/

**Comment by Sqrlly** [comment]
If you're using the .env file there is also a HOST_PORT= setting in there.  See .env.example.  But as stated above, the command line option is probably the easiest.
